config/routes.rb

```diff
@@ -20,6 +20,7 @@
   get "/register", to: "registration#index", as: :registration
   get "/register/auth", to: "registration#authenticate", as: :registration_auth
   post "/user/registered_email", to: "spree/users#registered_email"
+  get "/user/set_lang/:lang", to: "application#set_lang", :as => :set_lang
```

Comment:
Should this maybe be a `post`? It is changing something, but not on the server, since it just changes the cookie. I think it's okay to test/merge as it is though.

Reply:
If I set to a post here, then I can't handle it by a simple link in the langage selection dropdown. I must do a post request which is not as simple as a get (with a link). That's why, I prefer the `get` method. 
Let me know if it's a blocker for you, I'd happy to change my opinion.

Reply:
Yep, not a blocker, I think a `get` request makes sense.